src/applications/vaos/appointment-list/pages/CancelAppointmentPage/CancelConfirmationPage.jsx

```diff
@@ -29,16 +31,19 @@ function handleClick(history, dispatch, url) {
 export default function CancelConfirmationPage({ appointment, cancelInfo }) {
   const dispatch = useDispatch();
   const history = useHistory();
+  const useFeSourceOfTruth = useSelector(state =>
+    selectFeatureFeSourceOfTruth(state),
+  );
   const { typeOfCare: page } = useSelector(getNewAppointmentFlow);
   const { showCancelModal } = cancelInfo;
   const type = selectAppointmentType(appointment);
+  const isRequest = useFeSourceOfTruth
+    ? appointment.vaos.isPendingAppointment
```

Comment:
nit: do we need the `isPendingAppointment: isRequest,` line in our transformer if we can just rely on the new `pending` field?

Reply:
I had the same thought for all of `appointment.vaos.isPendingAppointment/isUpcomingAppointment/isPastAppointment`. I think it's redundant but I was hesitant to do that refactoring since these changes are already quite extensive, especially for reviewing. I think once we have validated these changes and have removed the feature flag to use the new logic, we would have a more reasonable time of refactoring and reducing the redundancy as a tech-debt work item.

Reply:
That is a good question. We also have 'isUpcomingAppointment` and `isPastAppointment` in the transformer we can cleanup in a later ticket.